### ISSUE TITLE
Nothing visible and impossible to save files

### ISSUE THREAD

**Issue by Roellen** [thunderbird-conversations/thunderbird-conversations#1769]
Hi. 

I have a problem using Conversations 4.0.8 with Thunderbird 91.4.1 (32-Bit) in Windows 10 64Bit.

1. Issue:
After activating the extension every email just shows a blank white screen. 
Only when double clicking a mail everything is shown in the new window.

When I deactiave and re-activate Conversation, then it works for a while. 

2. Issu:
When Conversations is activated I can't drag and drop any file attachments. It just saves a "gtk-file" of 3KB (seems to be the file-icon)

I can make a video if that helps? I really love Conversations, without it I would need to change the mail program. 

Thanks.

**Comment by Standard8** [comment]
When it stops working, can you see if there's any messages (especially errors) on the error console? It is available from the three-bar application menu -> Tools -> Developer Tools -> Error Console.

Drag and drop is #1722. Unfortunately not an easy issue to resolve at the moment.

**Comment by Standard8** [comment]
@Roellen did you see my previous comment?

**Comment by Roellen** [comment]
Hi. Sorry for not replying earlier. Here is the error-log:

```
12:47:48.970 Unexpected event profile-after-change URLQueryStrippingListService.jsm:224
12:47:50.823 [l10nregistry] Attempting to synchronously load file
            resource:///chrome/en-US/locale/en-US/calendar/messenger/otr/otrUI.ftl while it's being loaded asynchronously. L10nRegistry.jsm:597:19
12:47:50.824 [Exception... "Component returned failure code: 0x80520001 (NS_ERROR_FILE_UNRECOGNIZED_PATH) [nsIXPCComponents_Utils.readUTF8URI]"  nsresult: "0x80520001 (NS_ERROR_FILE_UNRECOGNIZED_PATH)"  location: "JS frame :: resource://gre/modules/L10nRegistry.jsm :: L10nRegistry.loadSync :: line 692"  data: no] L10nRegistry.jsm:692:19
12:47:50.825 Trying to load C:\Program Files (x86)\Mozilla Thunderbird\libotr.dll OTRLib.jsm:64:11
12:47:50.876 Successfully loaded OTR library C:\Program Files (x86)\Mozilla Thunderbird\libotr.dll OTRLib.jsm:72:13
12:47:50.942 Successfully loaded OpenPGP library rnp.dll version 0.15.2+git20210806.dd923a4e.MZLA from C:\Program Files (x86)\Mozilla Thunderbird\rnp.dll RNPLib.jsm:92:15
12:47:51.240 Found 0 public keys and 0 secret keys (0 protected, 0 unprotected) RNPLib.jsm:288:15
12:47:51.456 tb.account.size_on_disk - Truncating float/double number. 8
12:47:51.457 tb.account.size_on_disk - Trying to set an unsigned scalar to a negative number. 2
12:47:51.923 Unknown Collection "thunderbird/query-stripping" RemoteSettingsClient.jsm:160
12:52:19.002 1644753139002	addons.xpi-utils	WARN	disabling legacy extension tbthemefixed@wanhunglo.com
12:53:02.320 Einige Cookies verwenden das empfohlene "SameSite"-Attribut inkorrekt. 2
12:54:19.861 services.settings: thunderbird/hijack-blocklists has signature disabled RemoteSettingsClient.jsm:1027
12:55:37.440 Uncaught (in promise) Error: Error while loading 'jar:file:///C:/Users/Frank/AppData/Roaming/Thunderbird/Profiles/cevsgqcp.Frank%202019/extensions/tbthemefixed@wanhunglo.com.xpi!/manifest.json' (NS_ERROR_FILE_NOT_FOUND)
    readJSON resource://gre/modules/Extension.jsm:655
    onStopRequest resource://gre/modules/NetUtil.jsm:128
Extension.jsm:655:20
12:55:37.698 Uncaught (in promise) Error: Error while loading 'jar:file:///C:/Users/Frank/AppData/Roaming/Thunderbird/Profiles/cevsgqcp.Frank%202019/extensions/tbthemefixed@wanhunglo.com.xpi!/manifest.json' (NS_ERROR_FILE_NOT_FOUND)
    readJSON resource://gre/modules/Extension.jsm:655
    onStopRequest resource://gre/modules/NetUtil.jsm:128
Extension.jsm:655:20
12:55:42.160 Error: Can't find profile directory. 2 XULStore.jsm:66:15
12:56:23.301 Conversations.Conversation: Warning: gloda query returned no messages conversation.js:230:17
12:56:23.302 Conversations.Conversation: Streaming the message because Gloda has not indexed it, this will be slower conversation.js:123:7
12:56:23.651 TypeError: from is undefined
    MessageHeader chrome://conversations/content/902.bundle.js:3018
    Ch chrome://conversations/content/465.bundle.js:373
    ck chrome://conversations/content/465.bundle.js:483
    bk chrome://conversations/content/465.bundle.js:466
    ak chrome://conversations/content/465.bundle.js:466
    Tj chrome://conversations/content/465.bundle.js:466
    Lj chrome://conversations/content/465.bundle.js:459
    jg chrome://conversations/content/465.bundle.js:339
    unstable_runWithPriority chrome://conversations/content/465.bundle.js:570
    gg chrome://conversations/content/465.bundle.js:338
    jg chrome://conversations/content/465.bundle.js:339
    ig chrome://conversations/content/465.bundle.js:338
    Wj chrome://conversations/content/465.bundle.js:460
    notify chrome://conversations/content/367.bundle.js:366
    notifyNestedSubs chrome://conversations/content/367.bundle.js:436
    handleChangeWrapper chrome://conversations/content/367.bundle.js:441
    dispatch chrome://conversations/content/367.bundle.js:970
    middleware chrome://conversations/content/367.bundle.js:590
    dispatch chrome://conversations/content/367.bundle.js:1320
    updateConversation chrome://conversations/content/902.bundle.js:1522
465.bundle.js:432:200
12:56:23.656 Uncaught (in promise) TypeError: from is undefined
    MessageHeader chrome://conversations/content/902.bundle.js:3018
    Ch chrome://conversations/content/465.bundle.js:373
    ck chrome://conversations/content/465.bundle.js:483
    bk chrome://conversations/content/465.bundle.js:466
    ak chrome://conversations/content/465.bundle.js:466
    Tj chrome://conversations/content/465.bundle.js:466
    Lj chrome://conversations/content/465.bundle.js:459
    jg chrome://conversations/content/465.bundle.js:339
    unstable_runWithPriority chrome://conversations/content/465.bundle.js:570
    gg chrome://conversations/content/465.bundle.js:338
    jg chrome://conversations/content/465.bundle.js:339
    ig chrome://conversations/content/465.bundle.js:338
    Wj chrome://conversations/content/465.bundle.js:460
    notify chrome://conversations/content/367.bundle.js:366
    notifyNestedSubs chrome://conversations/content/367.bundle.js:436
    handleChangeWrapper chrome://conversations/content/367.bundle.js:441
    dispatch chrome://conversations/content/367.bundle.js:970
    middleware chrome://conversations/content/367.bundle.js:590
    dispatch chrome://conversations/content/367.bundle.js:1320
    updateConversation chrome://conversations/content/902.bundle.js:1522
902.bundle.js:3018:5
12:56:37.812 Conversations.Conversation: Warning: gloda query returned no messages conversation.js:230:17
12:56:37.813 Conversations.Conversation: Streaming the message because Gloda has not indexed it, this will be slower conversation.js:123:7
12:56:48.476 Conversations.Conversation: Warning: gloda query returned no messages conversation.js:230:17
12:56:48.477 Conversations.Conversation: Streaming the message because Gloda has not indexed it, this will be slower conversation.js:123:7

```

Thanks.

**Comment by Roellen** [comment]
Hmm. I think I never received a mail without a sender-adress. 
The problem is this? >>> "TypeError: from is undefined"

It also happens when I receive two mails from a sender who has a mail adress. 
When I deactivate conversations and avtivate it again... then it works for a short time. 
And sometimes when I click on the conversation, then the conversations are looking fine. 
But in the "normal window" of Thunderbird... it does not show anything. So I can't even read any mail. 

Not sure how to troubleshoot this. Maybe I can install it on another computer to see if it works there?

Thanks!

**Comment by Standard8** [comment]
Is this still an issue in the latest Conversations, 4.0.14?

**Comment by Roellen** [comment]
I'll check this and let you know. (I switches it off completely... but now switched it on to see if it works)
Thanks.

**Comment by Roellen** [comment]
Ah... I think I switched it off because moving files from mails to a folder did not worked anymore. But I need this for every mail. 

https://www.youtube.com/watch?v=73kh2mhPbog&ab_channel=FrankR%C3%B6llen

**Comment by Standard8** [comment]
I don't understand what you mean by "I switched it off because moving files from mails to a folder did not worked anymore", that's not part of Conversations' functionality.

The video doesn't work - it says it has been removed.

**Comment by Roellen** [comment]
Hi. 
I had to remove the video, as it has shown the adress of a customer. (did not thought about that when doing the video)

When I switch on "conversations" then I can't drag and drop attached files from the email-window to a local folder. 
When I switch off "conversation" then it's working correctly. 

Instead of finding the attached file on my local folder, it always saves  a small file named "gtk-file". (which I already wrote previously)

So, to see if it's just a problem with my installation or computer I tried "conversations" on another computer. It's different on the other computer. When drag and drop a file to a local folder it's not "gtk-file" but it's "application-pdf" which is saved. And this does not have a *.pdf extension. 
I changed the extension from svg to pdf (by using command line in admin mode and using "REN application-pdf.svg PDF.pdf" but the file does not open with acrobat reader. 

So I think it's maybe a general problem? 

I can make another video if it helps? Just let me know. 

Thanks!
